root/release/edit/information.tt

```diff
@@ -197,7 +197,7 @@
           </span>
         </td>
         [% table_cells_select('country', add_colon(l('Country')), 1,
-            "value: countryID, withLabel: 'country'", countries) %]
+            "value: countryID, withLabel: 'country', controlsBubble: \$root.countryBubble", countries) %]
```

Comment:
(I think the backslash is a no-op here, but may confuse people as to whether or not it's intentional :))
```suggestion
            "value: countryID, withLabel: 'country', controlsBubble: $root.countryBubble", countries) %]
```

Reply:
I don't understand why it's needed either, but without it, I get an unhelpful error on the JS console and a partially-rendered page:

```
Uncaught TypeError: Unable to process binding "with: function(){return rootField.release }"
Message: Unable to process binding "foreach: function(){return events }"
Message: Unable to process binding "controlsBubble: function(){return undefined }"
Message: Cannot read properties of undefined (reading 'group')
    at init (Bubble.js:180:40)
    ...
```

My best guess was that TT was trying to interpret it otherwise.

Reply:
You're right, I missed that this particular string is using double-quotes, while others are using single-quotes. Thanks for clarifying!